packages/shared/prisma/migrations/20230924232619_datasets_init/migration.sql

```diff
@@ -16,6 +16,7 @@ CREATE TABLE "dataset_items" (
     "expected_output" JSONB,
     "source_observation_id" TEXT,
     "dataset_id" TEXT NOT NULL,
+    "comment" TEXT,
```

Comment:
This will not create the comment field for users that already ran the migration. Instead you can execute the following in `packages/shared` to generate a new migration file based on the changes you've made in the `schema.prisma`:  `npx dotenv -e ../../.env -- npx prisma migrate dev --name your_migration_name`.